lib/utils.go

```diff
@@ -39,6 +39,11 @@ import (
 	. "github.com/status-im/status-go/testing" //nolint: golint
 )
 
+type jailFunc struct {
+	FuncRef func(*C.char, *C.char) *C.char
```

Comment:
I would suggest swapping the order of `jailFunc` so that `Name` goes first. It will be easier to reason about otherwise it suggests some params to function passed in the first field.